src/MCPClient/requirements/base.txt

```diff
@@ -0,0 +1,11 @@
+bagit==1.5.2
+Django>=1.7,<1.8
+django-annoying==0.7.7
+django-autoslug==1.7.1
+django-mysqlpool==0.1-9
+django-extensions==1.1.1
+mysqlclient==1.3.7
```

Comment:
When adding this requirement to the MCPClient , shouldn't mysql-client also be added to the debian/control file?  If MCPClient and the dashboard are both installed on the same machine, then it won't be a problem, but I think it would be good to add the os dependency in the same PR as the pip requirement.